### ISSUE TITLE
Feature: Show author and latest change date

### ISSUE THREAD

**Issue by toschdev** [cosmos/vuepress-theme-cosmos#254]
Since the vuepress docs theme gets the content from GitHub, it would be great to have with each article a quick intro into who initially created the page and when, who latest edited it, and when. This would help to spot outdated content, as well as reach the author when something is unclear.

**Comment by alijnmerchant21** [comment]
This is a good idea. I'm totally supportive of it!

**Comment by barriebyron** [comment]
yes! This approach also adds a nice human touch to our technical content.

**Comment by soniasingla** [comment]
I love this feature, more supportive, easy to approach. On this note, we should focus on automating too, if someone makes any changes to any tutorials, the person’s name must be automatically reflected in the editor section. Otherwise, maintaining this is huge chore! :)
